Change to namespace package (#2393)
certtools/intelmq: docs/dev/guide.rst

 * 3: statistics
 * 4: tests

Add a new Bot
=============

The ``intelmq/bots`` folder is defined as a `python namespace package <https://packaging.python.org/en/latest/guides/packaging-namespace-packages/>`_, 
thus you can add a package with your own bots (in this example ``additional_bots``). This package must contain the following structure:

.. code-block:: text

   additional_bots
   ├── setup.py
   └── intelmq
        ├── __init__.py
        └── bots
             └── collectors
                  └── your_bot
                       └── your_bot.py

The ``additional_bots`` folder needs to be defined e.g. with a ``setup.py``. Only the intelmq folder contains a ``__init__.py`` 

> file and it must contain only this line. Any additional code in ``__init__.py`` after this line will be inaccessible.

Reviewer: ```suggestion
file, which must have only the following line. Any additional code in ``__init__.py`` after this line will be inaccessible.
```
